chains/evm/cli/initialize/initialize.go

```diff
@@ -2,13 +2,14 @@ package initialize
 
 import (
 	"fmt"
+	"math/big"
+
 	"github.com/ChainSafe/chainbridge-core/chains/evm/calls"
 	"github.com/ChainSafe/chainbridge-core/chains/evm/calls/evmclient"
 	evmgaspricer "github.com/ChainSafe/chainbridge-core/chains/evm/calls/evmgaspricer"
 	"github.com/ChainSafe/chainbridge-core/chains/evm/calls/transactor"
 	"github.com/ChainSafe/chainbridge-core/crypto/secp256k1"
 	"github.com/rs/zerolog/log"
-	"math/big"
 )
 
 func InitializeClient(
```

Comment:
Please add godoc for this method, make sure that purpose of prepare argument is explicitly described